README.md

```diff
@@ -104,6 +104,10 @@ Search links that point directly to suitable issues to contribute to on GitHub.
 - [How to get started with Open Source - By Sayan Chowdhury](https://www.hackerearth.com/getstarted-opensource/)
 - [What open-source should I contribute to](https://kentcdodds.com/blog/what-open-source-project-should-i-contribute-to/) by Kent C. Dodds
 - [An immersive introductory guide to Open-source](https://developeraspire.hashnode.dev/an-immersive-introductory-guide-to-open-source) by [Franklin Okolie](https://twitter.com/DeveloperAspire)
+- [Getting started with contributing to open source](https://stackoverflow.blog/2020/08/03/getting-started-with-contributing-to-open-source/) by [Zara Cooper](https://stackoverflow.blog/author/zara-cooper/)
+- [Beginner's guide to open-source contribution](https://workat.tech/general/article/open-source-contribution-guide-xmhf1k601vdj) by [Sudipto Ghosh](https://github.com/pydevsg)
+- [Open source projects on GitHub that need your help.](https://www.codetriage.com/) by [codetriage](https://www.codetriage.com/)
```

Comment:
This is already in the first section, so it can be removed.

```suggestion
```

Reply:
@erictleung Sure I will do that...

Reply:
@erictleung I have removed that can you check now